src/rgw/rgw_multi.cc

```diff
@@ -248,6 +249,7 @@ int abort_multipart_upload(RGWRados *store, CephContext *cct,
           remove_objs.push_back(key);
         }
       }
+      parts_accounted_size += obj_part.size;
```

Comment:
should use obj_part.accounted_size since we use accounted_size when add the quota, it's the size before compression

Reply:
Thank you for your review. Please help to review it again
issue: https://tracker.ceph.com/issues/41606